schematic/schemas/data_model_graph.py

```diff
@@ -956,7 +954,10 @@ def get_node_column_type(
         """
         node_label = self._get_node_label(node_label, node_display_name)
         rel_node_label = self.dmr.get_relationship_value("columnType", "node_label")
-        return self.graph.nodes[node_label][rel_node_label]
+        type_string = self.graph.nodes[node_label][rel_node_label]
+        if type_string is None:
+            return type_string
+        return JSONSchemaType(type_string)
 
```

Comment:
This is where the bug was. This was returning the type as a string, btu should ahve been returning it as the enum.